obspy/io/seiscomp/inventory.py

```diff
@@ -444,14 +455,23 @@ def _read_channel(instrumentation_register, cha_element, _ns):
                 response_elements.append(found_response)
 
         if len(response_elements) == 0:
-            msg = ("Could not find response tag with public ID "
-                   "'{}'.".format(response_id))
-            raise obspy.ObsPyException(msg)
+            msg = ("Found multiple matching response tags with the same "
+                   "public ID '{response}'. "
```

Comment:
You got hit by copy/pasta there ;-)

Reply:
Ah yes, the first warning message ``"Could not find response tag with public ID '{response}'"`` has been restored now for when ``len(response_elements) == 0``